app/controllers/users_controller.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+class UsersController < ApplicationController
+  around_action :unscope_resources
+  load_resource :user
+
+  def show
+    @course_users = @user.course_users.with_course_statistics
+    @current_course_users = @course_users.from_instance(current_tenant).merge(Course.current)
```

Comment:
Only the course_users in current tenant should be displayed to normal users, all course_users should be displayed to admins.

Also need to eager load courses associated with the user.